### PR TITLE
[grafana] bump kiwigrid/k8s-sidecar version to 1.26.0

### PR DESCRIPTION
**Pull request by kvanzuijlen**
https://github.com/kiwigrid/k8s-sidecar/compare/1.25.2...1.25.6

### CODE REVIEW

**Review by zanhsieh — APPROVED** [review]

**Review by Xtigyro — APPROVED** [review]

**Comment by kvanzuijlen** [comment]
@zanhsieh In the meantime 1.26.1 was released. Should I update to 1.26.1 in this PR?

**Comment by zanhsieh** [comment]
@kvanzuijlen 
I think we better stick on the approved version since I am not sure it would need another maintainers approvals again.

**Pull request merged**
